code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -23,9 +23,16 @@
 	if(prob(getarmor(hit_zone, ARMOR_MELEE))) //melee armour provides a chance to turn sharp/edge weapon attacks into blunt ones
 		weapon_sharp = 0
 		weapon_edge = 0
+		for(var/obj/item/clothing/suit/space/SS in src.get_equipped_items()) //checks if we're wearing a space suit
```

Comment:
Having special check for space suits in mob code is bad. If breaching system need more work, it should be done as part of the proper armor rework instead.